docs/diagrams/web.md

```diff
@@ -0,0 +1,62 @@
+```mermaid
+flowchart TB
+  direction BT
+  %% Styles
+  classDef userStyle stroke:#167CC5,stroke-width:4px
+  classDef processStyle stroke:#bc0061,stroke-width:4px
+  classDef saasStyle stroke:#f2ff07,stroke-width:4px
+  classDef dbStyle stroke:#010daf,stroke-width:4px
+
+  %% Component Defintions
+
+  %% Users
+  user_unprivleged["👤  <br/> Unprivileged User <br/> *cloud.gov Customer"]:::userStyle
+
+  %% SaaS
+  dap[Analytics <br/> *DAP]:::saasStyle
+
+  %% Processors
+  auth_provider[Authentication Provider <br/> *UAA]:::processStyle
+  dashboard_app[Dashboard Web Application <br/> *Next.js]:::processStyle
+
+  %% Databases
+  database[(Database <br/> *AWS RDS Postgres)]:::dbStyle
+  s3_bucket[(Object storage<br/> *AWS S3 Bucket)]:::dbStyle
+
+  %% Flow
+  auth_provider -- All Users <br/> HTTPS Port 443--> dashboard_app
+
+  user_unprivleged -- Request Content <br/> HTTPS Port 443 --> auth_provider
+  user_unprivleged -- Reports Usage <br/> HTTPS Port 443 --> dap
+
+
+  dashboard_app -- Read/Write Application Data <br /> Authenticated TLS Port 5432  --> database
+  dashboard_app -- Read/Write UAA Bot Storage <br /> Authenticated TLS Port 6379  --> s3_bucket
```

Comment:
[non-blocking question] Where did you find the port for s3?  I'm curious just because I hadn't come across it anywhere yet and I'm not sure where to find that information!

Reply:
Good catch, I think `5432` is left over from Postgres RDS that was in the diagram previously. As far as I can tell communicating with S3 happens on 80/443, so I'll update it to that. TY!

Reply:
Oh cool, I figured I was just missing something.  The 5432 might still be accurate for the database, but yeah 443 matches up with what I could find for S3.

Reply:
Oops sorry, yeah `5432` is the port for Postgres, meant to say the `6379` was the port for Redis that was in the previous Pages diagram.